Source/WebKit/UIProcess/RemotePageProxy.cpp

```diff
@@ -110,19 +110,20 @@ void RemotePageProxy::injectPageIntoNewProcess()
 
 void RemotePageProxy::processDidTerminate(WebProcessProxy& process, ProcessTerminationReason reason)
 {
-    if (!m_page)
+    RefPtr page = m_page.get();
+    if (!page)
         return;
-    if (auto* drawingArea = m_page->drawingArea())
+    if (auto* drawingArea = page->drawingArea())
```

Comment:
Why not use CheckedPtr here?

Reply:
The static analysis didn't ask for it, I only fixed the issues that were reported by the analysis.

Reply:
Looks like this may be a false negative?